### PR TITLE
Add LddAllowUnreachable

### PR DESCRIPTION
**Pull request by ling2yt**

### CODE REVIEW

File: pkg/ldd/ldso_linux.go [hunk 1]
```diff
@@ -11,6 +11,9 @@ import (
 
 // LdSo finds the loader binary.
 func LdSo(bit64 bool) (string, error) {
+	if os.Getenv("LD_LIB_PATH") != "" {
```
Comment (rminnich) [inline pkg/ldd/ldso_linux.go:14]: This name is wrong.

**Review by rminnich — CHANGES_REQUESTED** [review]
There's a lot of things in here that don't compile. You will need to fix that first.

**Comment by ling2yt** [comment]
create the pull request by mistake

**Comment by ling2yt** [comment]
> There's a lot of things in here that don't compile. You will need to fix that first.

Sorry I create the pull request by mistake. And what I want to confirm is whether we will support call the cross compile ldd to resolve the dependencies like find the arm execution dependencies on the x86 platform?

**Pull request closed without merging**